contracts/test/warp.ts

```diff
@@ -0,0 +1,49 @@
+// (c) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import {
+  Contract,
+} from "ethers"
+import { ethers } from "hardhat"
+
+const WARP_ADDRESS = "0x0200000000000000000000000000000000000005";
+var senderAddress: string
+// Expected to be a hex string
+var payload: string
+var expectedUnsignedMessage: string
+var sourceID: string
+
+describe("IWarpMessenger", function () {
+  this.timeout("30s")
+
+  let owner: SignerWithAddress
+  let contract: Contract
+  before(async function () {
+    senderAddress = process.env["SENDER_ADDRESS"];
+    owner = await ethers.getSigner(senderAddress);
+    contract = await ethers.getContractAt("IWarpMessenger", WARP_ADDRESS, owner)
+
+    payload = process.env["PAYLOAD"];
+    expectedUnsignedMessage = process.env["EXPECTED_UNSIGNED_MESSAGE"];
+    sourceID = process.env["SOURCE_CHAIN_ID"];
```

Comment:
Can we move these variables to be set on lines 12-16 instead of in the before suite?